exercises/binary_converter.cpp

```diff
@@ -5,3 +5,40 @@
   Insert first number: 8
   The binary number is: 1000
 */
+
+
+#include<iostream>
+#include<string>
+#include<sstream>
+
+using namespace std;
+
+
+string toBinary(int num) {
+  //stringstream ss;
+  string bin;
+
+  if(num/2 == 0) {
+    bin = to_string(num%2);
+    return bin;
```

Comment:
```c++
    return to_string(num%2);
```
in realtà potresti anche lasciarlo così, perchè aumenta la leggibilità usare la variabile `bin`, ma ti scrivo lo stesso questa alternativa